WPMedia/fusion-news-theme-blocks
# THEMES-1195 Updated headline, overline and image opacity

## Description

Review doc: https://paper.dropbox.com/doc/Themes-hover-states-audit--B400R8DxsjiVy0YImyMBZ0B4Ag-JyHvxHuZttlIfj5BzjTI3

Redesigned button components: https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=7736%3A104212&t=wkfbbYPc5dtqGGKM-1 - Connect your Figma account 

## Jira Ticket

- [THEMES-1195](https://arcpublishing.atlassian.net/browse/THEMES-1195)

## Acceptance Criteria

Applies to all promos (manual and non, S-XL):

1. image when link:

   a. opacity change to 80%

2. overline (only on XL, L):

    a. rest: $text-color

    b. hover: $text-color-subtle

3. headline: 

     a. rest: $text-color

     b. hover: $text-color-subtle

4. author name, when link (covered in linked ticket so no need to override but confirm as part of this ticket):

     a. rest: $text-color

     b. hover: $text-color-subtle

## Test Steps

1. Checkout this branch `git checkout THEMES-1195`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/extra-large-manual-promo-block,@wpmedia/extra-large-promo-block,@wpmedia/large-manual-promo-block,@wpmedia/large-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/medium-promo-block,@wpmedia/small-manual-promo-block,@wpmedia/small-promo-block`
3. add all the blocks and inspect on hover to check the color

## Dependencies or Side Effects

- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/431
- Components pr. for local development: https://github.com/WPMedia/arc-themes-components/pull/199

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1195]: https://arcpublishing.atlassian.net/browse/THEMES-1195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
